tests/web3js/build_evm_state_test.js

```diff
@@ -156,7 +156,7 @@ it('should handle a large number of EVM interactions', async () => {
         gas: 55_000,
         gasPrice: conf.minGasPrice
     }, 82n)
-    assert.equal(estimatedGas, 21358n)
+    assert.equal(estimatedGas, 21646n)
```

Comment:
Change comes from: `EIP-7623: Increase calldata cost`.
See: https://mixbytes.io/blog/the-prague-electra-pectra-hardfork-explained#rec860520137 .